src/tests/suite/daos_rebuild_ec.c

```diff
@@ -271,7 +271,7 @@ rebuild_ec_setup(void  **state, int number, uint32_t rf)
 		print_message("It can not create the pool with %d ranks"
 			      " probably due to not enough ranks %d\n",
 			      number, rc);
-		return 0;
+		return rc;
```

Comment:
We should remove the comment above that says
```
		/* Let's skip for this case, since it is possible there
		 * is not enough ranks here.
		 */
```